itests/docstest1.java

```diff
@@ -0,0 +1,10 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//DOCS readme.md
+
+import static java.lang.System.*;
+
+public class docstest1 {
+    public static void main(String... args) {
+        out.println("Docs test 1"));
```

Comment:
all the tests has a ")" too much on this line resulting in compile/build errors if trying to use them. intentional?

Reply:
I thing it's a typo